client/client_app/src/bookdetails.js

```diff
@@ -2,34 +2,48 @@ import React, { Component } from "react";
 import './App.css';
 import booksFromDB from "./load_books"
 import PrismaZoom from 'react-prismazoom';
+import Comments  from './comments'
```

Comment:
I think you forgot the add a file because the comments.js file doesn't seem to exist

Reply:
I believe I added it now. I don't know what happened.